keyboards/kinesis/keymaps/kzar/rules.mk

```diff
@@ -0,0 +1,17 @@
+# Build Options
```

Comment:
Remove everything that matches keyboards/kinesis/rules.mk, remove all comments. (From https://github.com/qmk/qmk_firmware/pull/12430)

Reply:
Done.